browser-extension/addTournesolRecommendations.css

```diff
@@ -28,25 +28,74 @@
 
 .language_option {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   float: right;
   margin-top: 3px;
-  margin-left: 4px;
+  margin-left: 2px;
 }
 
 #tournesol_link {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   position: absolute;
   right: 0px;
   top: 15px;
 }
 
+#refresh_button {
+  float: right;
+  margin-top: 0px;
+  margin-left: 12px;
+  margin-right: 12px;
+}
+
 #tournesol_icon {
   position: absolute;
   left: 0px;
 }
 
+.icon {
```

Comment:
Similarly, `icon` is a very generic name for a class. It may conflict with other items on the page, or items injected by other extensions.  

If I understand correctly, this class is applied on interactive icons. Something like `.tournesol_button_icon` could be more appropriate.